aten/src/ATen/native/native_functions.yaml

```diff
@@ -4549,7 +4549,7 @@
   variants: function
 
 - func: where.ScalarOther(Tensor condition, Tensor self, Scalar other) -> Tensor
-  variants: function
+  variants: function, method
```

Comment:
Let's not extend this to be a method in this PR

Reply:
Without this the newly added `cases_scalar` won't work for the method variant (for OpInfo tests). Also without this the function seems half baked.

```python
x.where(cond, y) # works
x.where(cond, 1) # doesn't work
```